packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +61,313 @@ export default {
-          },
-          x: {
-            type: 'attribute',
-            name: 'disabled',
-            values: [false, true]
-          }
-        },
-        constants: [
-          clearableConstant,
-          multipleConstant,
-          fiveOptionsConstant,
-          {
-            type: 'attribute',
-            name: 'value',
-            value: 'option-1 option-2 option-3 option-4'
-          }
-        ],
-        args
-      })}
-    </div>`;
-  }
+export const Placement = {
+  render: () => html`
+    <div class="flex items-center gap-12 h-[500px]">
+      <sd-select class="self-baseline" size="lg" label="Label" placement="bottom" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+
+      <sd-select size="lg" label="Label" placement="top" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * `sd-select` with valid and invalid styles.
+ * Use the `label` attribute to provide an accessible label for the select element.<br />Use the `label` slot to add custom markup.
  */
 
-export const ValidInvalid = {
-  name: 'Valid x Invalid',
-  parameters: {
-    controls: {
-      exclude: ['label', 'open-attr', 'required', 'default', 'useTags', 'multiple', 'max-options-visible']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<form class="h-[260px] w-full flex gap-4">
-      ${generateTemplate({
-        options: {
-          classes: 'w-full [&>tbody>tr>td]:align-top'
-        },
-        axis: {
-          y: {
-            type: 'attribute',
-            name: 'useTags',
-            values: [false, true]
-          },
-          x: {
-            type: 'attribute',
-            name: 'value',
-            values: ['option-1 option-2', '']
-          }
-        },
-        constants: [
-          twoOptionsConstant,
-          labelConstant,
-          multipleConstant,
-          { type: 'attribute', name: 'required', value: true }
-        ],
-        args
-      })}
-      <sd-button class="hidden" type="submit">Submit</sd-button>
-    </form>`;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('sd-button');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const Label = {
```

Comment:
Pls align with what’s happening in sd-input right now
[ * - [Select with tooltip](?path=/docs/templates-select--docs#select%20with%20tooltip)
](https://solid-design-system.github.io/solid/feature_sd-input-docs/?path=/docs/components-sd-input--docs#label)
<img width="1035" alt="Bildschirmfoto 2024-09-11 um 13 51 20" src="https://github.com/user-attachments/assets/9a925188-291d-452f-ab24-0c54442cfe89">

Reply:
Needs to be also mirrored to sd-textarea

Reply:
> Pls align with what’s happening in sd-input right now [ * - [Select with tooltip](?path=/docs/templates-select--docs#select%20with%20tooltip) ](https://solid-design-system.github.io/solid/feature_sd-input-docs/?path=/docs/components-sd-input--docs#label) <img alt="Bildschirmfoto 2024-09-11 um 13 51 20" width="1035" src="https://private-user-images.githubusercontent.com/41894356/366443677-9a925188-291d-452f-ab24-0c54442cfe89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYwNTYwNjAsIm5iZiI6MTcyNjA1NTc2MCwicGF0aCI6Ii80MTg5NDM1Ni8zNjY0NDM2NzctOWE5MjUxODgtMjkxZC00NTJmLWFiMjQtMGM1NDQ0MmNmZTg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTExVDExNTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxMTI2MWIzOTYyZTViOTI0ODNhNzYwNjkzNjI2MjhiYjQxOTI3ZmNhNjQ3YTcwM2M0Y2UyZTAxMzVjZWEzMzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L3lp_oPqajt6xbqBkOVZXCzYpGVfMixRjtgDkUY5fQg">

Updated

Reply:
> Needs to be also mirrored to sd-textarea

Addressed [#1367](https://github.com/solid-design-system/solid/pull/1367)